storage/statedb/database.go

```diff
@@ -98,6 +98,7 @@ type Database struct {
 	gctime  time.Duration      // Time spent on garbage collection since last commit
 	gcnodes uint64             // Nodes garbage collected since last commit
 	gcsize  common.StorageSize // Data storage garbage collected since last commit
+	gcLock  sync.RWMutex       // Lock for preventing to garbage collect cachedNode without flushing.
```

Comment:
Minor suggestion, but just for consistency.
```suggestion
	gcLock  sync.RWMutex       // Lock for preventing to garbage collect cachedNode without flushing
```